pkg/artifact/sbom/cyclonedx_test.go

```diff
@@ -0,0 +1,301 @@
+package sbom_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/aquasecurity/fanal/artifact"
+	"github.com/aquasecurity/fanal/cache"
+	"github.com/aquasecurity/fanal/types"
+	"github.com/aquasecurity/trivy/pkg/artifact/sbom"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestArtifact_Inspect(t *testing.T) {
```

Comment:
This test can be simpler. Looks like most cases are relevant to parsing logic. They should be tested in `pkg/sbom/cyclonedx`.